File: catalog/admin/section/delete.php
<?php
defined('AUTH') or die('Restricted access');

include_once $_SERVER['DOCUMENT_ROOT'].'/components/catalog/admin/classes/AdminCatalogSection.php';

$id = intval($_POST['id']);
if($id == 0 || $_POST['agree'] != 'yes')
	header('location: /admin/com/catalog');

$section = new AdminCatalogSection();
$section_data = $section->getSection($id);
$section->delete($id);

echo json_encode(array('answer' => 'success', 'catalog_id' => $section_data['catalog_id']));
exit;

File: dan/system/catalog/section/delete.py
import sys
import json
sys.path.append('system/catalog/classes')
from system.catalog.classes.Section import Section


def delete(SITE):
    print('FUNCTION -> system-> calalog -> section -> delete')

    id = SITE.post['id']

    SECTION = Section(SITE)

    if 'agree' in SITE.post and SITE.post['agree'] == 'yes':
        cat_id = SECTION.delete(id)
    else:
        cat_id = SECTION.getSection(id)['catalog_id']
    
    answer = {'answer': 'success', 'cat_id': cat_id}
    return {'ajax': json.dumps(answer)}